quickstart/dotnet-core/Program.cs

```diff
@@ -207,102 +208,97 @@ public static async Task ForceReconnectAsync()
-            Console.WriteLine("Cache response : " + cache.Execute(cacheCommand).ToString());
+            Console.WriteLine($"\nCache command: " + cacheCommand);
+            RedisResult pingResult = await BasicRetryAsync(async (db) => await db.ExecuteAsync(cacheCommand));
+            Console.WriteLine($"Cache response: {pingResult}");
 
             // Simple get and put of integral data types into the cache
             cacheCommand = "GET Message";
-            Console.WriteLine("\nCache command  : " + cacheCommand + " or StringGet()");
-            Console.WriteLine("Cache response : " + cache.StringGet("Message").ToString());
+            Console.WriteLine($"\nCache command: {cacheCommand} or StringGet()");
+            RedisValue getMessageResult = await BasicRetryAsync(async (db) => await db.StringGetAsync("Message"));
+            Console.WriteLine($"Cache response: {getMessageResult}");
 
             cacheCommand = "SET Message \"Hello! The cache is working from a .NET Core console app!\"";
-            Console.WriteLine("\nCache command  : " + cacheCommand + " or StringSet()");
-            Console.WriteLine("Cache response : " + cache.StringSet("Message", "Hello! The cache is working from a .NET Core console app!").ToString());
+            Console.WriteLine($"\nCache command: {cacheCommand} or StringSet()");
+            bool stringSetResult = await BasicRetryAsync(async (db) => await db.StringSetAsync("Message", "Hello! The cache is working from a .NET Core console app!"));
+            Console.WriteLine($"Cache response: {stringSetResult}");
 
-            // Demonstrate "SET Message" executed as expected...
             cacheCommand = "GET Message";
-            Console.WriteLine("\nCache command  : " + cacheCommand + " or StringGet()");
-            Console.WriteLine("Cache response : " + cache.StringGet("Message").ToString());
-
-            // Get the client list, useful to see if connection list is growing...
-            // Note that this requires allowAdmin=true in the connection string
-            cacheCommand = "CLIENT LIST";
-            Console.WriteLine("\nCache command  : " + cacheCommand);
-            var endpoint = (System.Net.DnsEndPoint)(await GetEndPointsAsync())[0];
-            IServer server = await GetServerAsync(endpoint.Host, endpoint.Port);
-            ClientInfo[] clients = await server.ClientListAsync();
-
-            Console.WriteLine("Cache response :");
-            foreach (ClientInfo client in clients)
-            {
-                Console.WriteLine(client.Raw);
-            }
+            Console.WriteLine($"\nCache command: {cacheCommand} or StringGet()");
+            getMessageResult = await BasicRetryAsync(async (db) => await db.StringGetAsync("Message"));
+            Console.WriteLine($"Cache response: {getMessageResult}");
 
             // Store .NET object to cache
             Employee e007 = new Employee("007", "Davide Columbo", 100);
```

Comment:
>Davide [](http://example.com/codeflow?start=49&length=6)

is "Davide" a typo?

Reply:
Hehe no I think it's an italian version of David! Will leave as-is